tailwind.config.js

```diff
@@ -99,6 +103,7 @@ module.exports = {
         "slide-out-right": "slide-out-right 200ms ease-in-out",
         "accordion-open": "accordion-open 200ms ease-in-out",
         "accordion-close": "accordion-close 200ms ease-in-out",
+        pulse: "pulse 1200ms cubic-bezier(0.4, 0, 0.6, 1) infinite",
```

Comment:
I just made this a touch slighter and adjusted the opacities to make it more obviously noticeable.